FBSnapshotTestCase/FBSnapshotTestCase.h

```diff
@@ -140,12 +140,17 @@
  - UIVisualEffect #70
  - UIAppearance #91
  - Size Classes #92
- 
- @attention If the view does't belong to a UIWindow, it will create one and add the view as a subview.
  */
 @property (readwrite, nonatomic, assign) BOOL usesDrawViewHierarchyInRect;
 
 /**
+ When YES, adds view to a window before capturing the snapshot.
+ @attention Can't be used with layers.
```

Comment:
a better semantics here would be "the setting only affects views, and ignored for layers", or "for layers this must be left to NO".